### ISSUE TITLE
Is the project discontinued?

### ISSUE THREAD

**Issue by BrunoMine** [MarkuBu/ambianceplus#1]
I want to help with the development, but I want to know if there is any prospect of continuation.

**Comment by MarkuBu** [comment]
Yes, all of my projects are discontinued. You can fork if you like and do whatever you want. But I moved to more real word projects and I'm learning wood working

**Comment by mckaygerhard** [comment]
there is two repos of ambience plus music .. what is the diffeence?
